app/controllers/api/api_controller.rb

```diff
@@ -1,4 +1,5 @@
 class Api::ApiController < ActionController::Base
+  include Rails::Pagination
```

Comment:
`paginate` now only automatically includes itself in `ActionController::API` – when I tried swapping the API controllers to that, they were unhappy about how we included Authentication and various other things (needs cookies, and needs to provide the `helper_method` function, and various other things). It says for applications where that's annoying, you can just include it manually like this.